moebot_bot/util/gifmaker.go

```diff
@@ -17,6 +17,7 @@ const (
 	xBorder   = 30
 	yBorder   = 14
 	maxWidth  = 500
+	viewDelay = 150 // 1/100th of second
```

Comment:
1/100th of a second?
I think you mean that it's in milliseconds, which would be a bit clearer. Not a big deal